src/tests/JIT/HardwareIntrinsics/XplatVirtualAlloc.cpp

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
we now have 3 copies of it, should we move this cpp file to some common location and consume it from cmake?

Reply:
I wouldn't mind having just one, but at the moment, I'm having trouble just getting this to work in general.